src/language_model/runner.py

```diff
@@ -0,0 +1,26 @@
+import os
+from importlib import import_module
+
+from .pipeline import SandboxTask, identifiers_from_config_file, init_logger
+
+TASK_FIELD_NAME: str = "task"
+
+
+class TaskRunner(object):
+    def run(self) -> None:
+        raise NotImplementedError()
+
+
+class ConfigurationFileRunner(TaskRunner):
+    def __init__(self, config_path: str):
+        self.config_path = config_path
+        module_name = os.path.splitext(config_path)[0].replace("/", ".")
+        module = import_module(module_name)
+        self.task: SandboxTask = getattr(module, TASK_FIELD_NAME)
```

Comment:
I'd suggest moving this to the run method or create another method e.g. `run_from_config`. 
In my understanding, the `__init__` method of the `ConfigurationFileRunner` should configure the runner itself. I.e. number of threads to use, logging, multiprocessing, etc...

Reply:
Following the "fail fast" concept, we initialize the maximum amount of variables as possible in the constructor method.
I see that part of initialization is implemented in `run` method. Need to review that to move everything to `__init__`.
What are the drawbacks of such an approach?

Reply:
The only drawback I see is that I find this combination of naming-behavior a bit counterintuitive. The `Runner` class to me sounds like a `Worker` in Celery - a thing that is created once and that accepts different tasks to run, the `Runner` initialization in this case should configure *how* we run the tasks, not which tasks we run. 
In the current implementation, the Runner is created per task and its task is set in the `__init__`. 

This class is responsible for obtaining the task via a path and supping it with essentially an output folder. Not exactly sure the `Runner` is a correct abstraction name here. I see it like we have a `TaskDescription` with the configuration and the commands that should be run to complete this task (`class Task` with `def execute`). And this `Runner` is more like a `TaskInstance` that should be run with a specific output folder. I do not really like the proposed naming, it's here to better convey the message.

I do see what you mean though and since we only run tasks one at a time and mostly via a terminal, this should work just fine.